.github/CONTRIBUTING.md

```diff
@@ -14,26 +14,35 @@ You can also open a [discussion](https://github.com/ScratchAddons/ScratchAddons/
 
 ## Contributing code
 
-If you are familiar with JavaScript, HTML, and CSS, you can contribute by fixing bugs, making adjustments, or adding features. Just follow the steps below to open a pull request to propose your changes which we can then add to Scratch Addons.
+If you are familiar with JavaScript, HTML, and CSS, you can contribute by fixing bugs or adding features. Just follow the steps below to open a pull request to propose your changes which we can then add to Scratch Addons.
 
 ### Learn more about making addons
 
 If you're making a new addon, our ["Creating an Addon" guide](https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/) will teach you how to make one.
 
 ### Opening a pull request
 
-We recommend using [Git](https://git-scm.com/) to commit changes. If you prefer GUIs, you can use [GitHub Desktop](https://desktop.github.com/) instead. If you don't want to install anything, you can use the GitHub website, but we recommend one of the other options because they can speed up some of the process.
+We recommend [using Git](https://docs.github.com/en/get-started/quickstart/set-up-git) to commit changes. If you prefer GUIs, you can also [set up GitHub Desktop](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/setting-up-github-desktop). If you don't want to install anything, you can use the GitHub website, but we strongly recommend one of the other options to speed up the process.
 
-If this is your first time contributing to a GitHub repository, you can practice by contributing to [octocat/Spoon-Knife](https://github.com/octocat/Spoon-Knife) with the help of [this guide](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
+New to contributing? You can practice by contributing to [octocat/Spoon-Knife](https://github.com/octocat/Spoon-Knife) with the help of [this guide](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
+
+Looking for a code editor? We recommend [Visual Studio Code](https://code.visualstudio.com/).
+
+We format our code with Prettier, which makes sure all code follows the same style and makes it look nice and neat. You can install [Prettier for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or enable the "Format code using Prettier" workflow on your fork to have Prettier format your code. If your pull request has formatting issues, don't worry, we can fix it for you.
```

Comment:
[This discussion](https://github.com/ScratchAddons/ScratchAddons/discussions/537#discussion-26269) explains this in further detail, so I might link to it.

Reply:
Hmm, that post is from 3 years ago

Reply:
There should be a way to put that on the docs...